code/online_challenges/src/project_euler/problem_022/problem_022.py

```diff
@@ -0,0 +1,16 @@
+def main():
+	with open('names.txt','r') as f:
+		name=f.read().strip('"').split('","')
```

Comment:
Add spaces around operators. Also, doesn't `.strip` remove the quotations so that only commas are left?